src/logger/console-logger.service.ts

```diff
@@ -29,43 +35,63 @@ export class ConsoleLoggerService {
       this.makeContextString(functionContext),
       false,
     );
-    this.printStackTrace(trace);
+    ConsoleLoggerService.printStackTrace(trace);
   }
 
   log(message: unknown, functionContext?: string): void {
-    this.printMessage(
-      message,
-      clc.green,
-      this.makeContextString(functionContext),
-      false,
-    );
+    if (
+      this.appConfig.loglevel === Loglevel.INFO ||
```

Comment:
Looking at these statements, I'm wondering why we don't define an order of the levels and just use something like `loglevel <= Loglevel.INFO`

Reply:
I build something similar with a helper function…